### ISSUE TITLE
[FEATURE REQUEST] Story arc support for Mylar-organized library?

### ISSUE THREAD

**Issue by cbarrerah** [beville/ComicStreamer#25]
Hi,
First of all, great work on this nifty piece of code, it was the cog that made all my comic-enjoying system work ^.^

I use Mylar to manage and organize my comic collection, and they have recently adde the option to manage story arcs (Convergence, Identity Crisis, etc...), would it be possible to add a new categorization in the database sorting to recognize them? 

That way, when accessing the comic collection from Chunky, i can dowload the whole story arc without the need to fetch individual comics.

Thanks in advance and keep being great!

**Comment by davide-romanini** [comment]
I don't know enough Mylar. I took a quick look and it seems basically a download manager for comics. If downloaded comics contain tags (in ComicRack's format for example) then story arcs should be already supported, at least at database level.

**Comment by cbarrerah** [comment]
Yes, the only addition needed is a "Story Arcs->" section in the interface presented when using the API. I'm talking about the database viewed from Chunky, for example.

**Comment by kurnalbert** [comment]
I have the exact same setup, comics downloaded, organised and tagged by Mylar and then read through Chunky. If a Story Arcs section was exposed through the API, it really would make this the most amazing way of streaming and reading comics!

**Comment by davide-romanini** [comment]
I'd be happy to add this feature, but I need some more detailed specification. Unfortunately I don't have Chunky for testing so I'm trying to guess.
The current api allows to filter per story arc with the `comiclist` endpoint, eg:

```
http://localhost:32500/comiclist?storyarc=test%
```

Storyarcs are also available from the `entities` endpoint, eg:

```
http://localhost:32500/entities/storyarcs/
```

So it seems the building blocks are already in place. Maybe is it possible that Chunky doesn't support it?

**Comment by wyldphyre** [comment]
Just thought I'd toss my 2 cents in, for what it's worth. Chunky has a free version now, and, in my experience, the developer is fairly responsive. You'll need an iPad though, which might be an issue.

**Comment by mike-ferenduros** [comment]
Yep, CS offers the APIs to do it, Chunky just doesn't support it right now. It's a super-easy addition, it just never occurred to me :-/

Should be in the next update.

PS: If there are any other requests for additions to the root menu, now's the time to ask!

**Comment by weidercs** [comment]
Sorry if it's not the right place, but along with story arc, maybe being able to re-order the comics in a group w/o having to rename them.  Took me a long time to figure out I could rename them with a number to put them in the order I needed them in.  Love your comic reader, use it every day!
